tests/unittests/test_handler/test_schema.py

```diff
@@ -507,9 +533,23 @@ def _get_schema_doc_examples():
 class TestSchemaDocExamples:
     schema = get_schema()
 
-    @pytest.mark.parametrize("example_path", _get_schema_doc_examples())
+    @pytest.mark.parametrize("example_path", _get_meta_doc_examples())
     @skipUnlessJsonSchema()
     def test_schema_doc_examples(self, example_path):
         validate_cloudconfig_file(str(example_path), self.schema)
 
+
+class TestStrictMetaschema:
+    '''Validate that schemas follow a stricter metaschema definition than
+    the default. This disallows arbitrary key/value pairs.
+    '''
+    def test_modules(self):
```

Comment:
If we don't have one, let's also add a unit test which asserts no invalid module-level schema raises a schema validation error when strict_metaschema=True. maybe with a pytest.mark.parametrize True,False on strict_metaschema to see validation error vs hiding it.